priv/riak_kv.schema

```diff
@@ -929,6 +929,12 @@
      (_) -> false
   end}.
 
+%% @doc Max heap size before dumping query buffers to ldb
+{mapping, "riak_kv.query.timeseries.qbuf_inmem_max", "riak_kv.timeseries_query_buffers_inmem_max", [
+  {default, "10MB"},
+  {datatype, bytesize}
```

Comment:
One more comment, sorry this didn't occur to me sooner. Ideally I think this would be a per-node setting, rather than per-query, but I think that can be tabled for a future release. However, the fact that this setting is per-query needs to be explicit in the comment in the schema file and in the docs.